core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -297,20 +297,6 @@ public void retry(String workflowId) throws Exception {
 
     }
 
-    public List<Workflow> getStatusByCorrelationId(String workflowName, String correlationId, boolean includeClosed) throws Exception {
```

Comment:
Did you intend to remove this method?

Reply:
Yes, it is not used anywhere else, and this functionality is replicated in ExecutionService.